startup_scripts/20_custom_fields.py

```diff
@@ -56,13 +56,14 @@ def get_class_for_class_path(class_path):
 
         custom_field.save()
 
-        for choice_details in cf_details.get('choices', []):
-          choice = CustomFieldChoice.objects.create(
+        for idx, choice_details in enumerate(cf_details.get('choices', [])):
+          choice, created = CustomFieldChoice.objects.get_or_create(
```

Comment:
You don't use the `created` variable. Is it a left-over or did you forget to use it?

Reply:
I can change that to `_` becuase it is not relevant to anything further down right now.